### ISSUE TITLE
ELB ping must be smarter

### ISSUE THREAD

**Issue by aerosol** [aerosol/kai#5]
Right now it only keeps-alive `erlang node <-> kairos` but doesn't `kairos <-> cassandra`.

Solution: periodically put metric with `timestamp=0` `value=now()` and `tag=node()`.

@loucash ?

**Comment by loucash** [comment]
If it is just for pings, than personally, I prefer to "read" which is less invasive.

**Comment by aerosol** [comment]
But reads go through a different port and won't keep the write socket alive

**Comment by aerosol** [comment]
So perhaps `version` for telnet ping and `read foobar` for cassandra ping?

**Comment by loucash** [comment]
awesome :+1:

**Comment by AeroNotix** [comment]
:+1: I approve of this message

**Comment by aerosol** [comment]
Pros for write: can query kairos for client pings / writes are cheaper?
Pros for read: separation of concerns, not polluting the DB with writes user may not care about, even if deploying on AWS

I'm conflicted ;-)

**Comment by loucash** [comment]
Exactly, that is why I wrote "if just for pings" -> read, "querying for kairos client pings" is a perfect reason for writes.

Let user decide, add option in configuration, implement both ;-)

**Comment by aerosol** [comment]
Shame there is no ying-yang emoji.

**Comment by aerosol** [comment]
Tentative start here https://github.com/aerosol/kai/pull/7

To be improved..
